packages/server/postgres/migrations/1611342655964_addOrganizationUserAuditTable.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`
+    CREATE TABLE "OrganizationUserAudit" (
+      id SERIAL PRIMARY KEY,
+      "orgId" VARCHAR(100) NOT NULL,
+      "userId" VARCHAR(100) NOT NULL,
+      "eventDate" TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
no timezone needed here, our DB & servers are all in UTC & if we need to convert it to a specific TZ we can, but we don't need to know the source tz.